pkg/es/config/config.go

```diff
@@ -33,48 +33,60 @@ import (
 	storageMetrics "github.com/jaegertracing/jaeger/storage/spanstore/metrics"
 )
 
+type TemplateOptions struct {
+	Priority    int `mapstructure:"priority"`
+	NumShards   int `mapstructure:"num_shards"`
+	NumReplicas int `mapstructure:"num_replicas"`
+}
+
+// IndexOptions describes the index format and rollover frequency
+type IndexOptions struct {
+	DateLayout          string `mapstructure:"date_layout"`
+	RolloverFrequency   string `mapstructure:"rollover_frequency"` // "hour" or "day"
+	TemplatePriority    int64  `mapstructure:"template_priority"`
+	TemplateNumShards   int64  `mapstructure:"template_num_shards"`
+	TemplateNumReplicas int64  `mapstructure:"template_num_replicas"`
```

Comment:
Do we need the word template here? I don't see what additional value it provides (unlike the priority field where it's relevant).